src/Console/ModelsCommand.php

```diff
@@ -1260,6 +1268,32 @@ protected function getClassKeyword(ReflectionClass $reflection)
         return $keyword;
     }
 
+    protected function checkForCastableCasts(string $type, array $params = [])
```

Comment:
I think we can safely define the proper return type
```suggestion
    protected function checkForCastableCasts(string $type, array $params = []): string
```